From c9dad8c75d2da05695199334c1d23ffe0744fd5f Mon Sep 17 00:00:00 2001
From: Philipp Zabel <p.zabel@pengutronix.de>
Date: Wed, 20 May 2015 11:34:08 +0200
Subject: [PATCH 0160/2508] drm/panel: simple: Add bus format for HannStar
 HSD100PXN1

commit 4946b0430c6933383d33adf101529b7085a4a682 upstream

This patch adds the bus_format field to the HSD100PXN1 panel structure.

Signed-off-by: Philipp Zabel <p.zabel@pengutronix.de>
Signed-off-by: Thierry Reding <treding@nvidia.com>
---
 drivers/gpu/drm/panel/panel-simple.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/drivers/gpu/drm/panel/panel-simple.c b/drivers/gpu/drm/panel/panel-simple.c
index 97b66b8..f94201b 100644
--- a/drivers/gpu/drm/panel/panel-simple.c
+++ b/drivers/gpu/drm/panel/panel-simple.c
@@ -752,6 +752,7 @@ static const struct panel_desc hannstar_hsd100pxn1 = {
 		.width = 203,
 		.height = 152,
 	},
+	.bus_format = MEDIA_BUS_FMT_RGB666_1X7X3_SPWG,
 };
 
 static const struct drm_display_mode hitachi_tx23d38vm0caa_mode = {
-- 
1.7.5.4

